components/agenda-fieldset/agenda-fieldset.js

```diff
@@ -0,0 +1,36 @@
+import { getLibs } from '../../scripts/utils.js';
+import { style } from './agenda-fieldset.css.js';
+import { convertTo24HourFormat } from '../../utils/utils.js';
+
+const { LitElement, html } = await import(`${getLibs()}/deps/lit-all.min.js`);
+
+export default class AgendaFieldset extends LitElement {
+  static properties = { timeSlots: { type: Array }, options: { type: Object } };
+
+  constructor() {
+    super();
+    this.timeSlots = this.dataset.timeSlots.split(',');
+    this.options = JSON.parse(this.dataset.options);
+  }
+
+  static styles = style;
+
+  render() {
+    return html`
+      <div class="field-container">
+        <div class="time-picker">
+          <p>Time</p>
+          <div class="time-picker-wrapper">
+            <sp-picker class="time-picker-input select-input" label="Pick agenda time">
+            ${this.timeSlots.map((timeSlot) => html`<sp-menu-item value=${convertTo24HourFormat(timeSlot)}">${timeSlot}</sp-menu-item>`)}
```

Comment:
nit: you can use repeat()

Reply:
Can I? Doesn't that repeat the same thing?

Reply:
https://github.com/adobecom/ecc-milo/pull/22/files#diff-7b19aecb9c630e6bb001202fb54caf392e71e6e2f192b2c0c5b4de18af05a38aR78